resources/views/masjid_profile/edit.blade.php

```diff
@@ -21,4 +31,110 @@
         </div>
     </div>
 </div>
+<div class="modal fade" id="modal-masjid" tabindex="-1" role="dialog" aria-labelledby="modalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="modalLabel">Laravel 10 Image Croper Upload Example - Laravelia </h5>
+          <button type="button" class="close" data-dismiss="modal-masjid" aria-label="Close">
+            <span aria-hidden="true">×</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <div class="img-container">
+              <div class="row">
+                  <div class="col-md-8">
+                      <img id="image" src="" alt="{{ Setting::get('masjid_name') ?? 'buku masjid'}}">
+                  </div>
+                  <div class="col-md-4">
+                      <div class="preview"></div>
+                  </div>
+              </div>
+          </div>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+          <button type="button" class="btn btn-primary" id="crop">Crop</button>
+        </div>
+      </div>
+    </div>
+</div>  
 @endsection
+
+@section('styles')
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.5.6/cropper.css"/>
+@endsection
+
+@push('scripts')
+<script src="https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.5.6/cropper.js"></script>
+<script>
+var $modal = $('#modal-masjid');
+var image = document.getElementById('image');
+var cropper;
+
+$(document).on("change", "#masjid_profile_image", function(e){
```

Comment:
Di sini kita ingin ubah logo, bukan profile image.

```suggestion
$(document).on("change", "#masjid_logo_image", function(e){
```